<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactsWindow</class>
 <widget class="QDialog" name="ContactsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>510</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>510</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="ContactsFunctionsBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>470</width>
     <height>440</height>
    </rect>
   </property>
   <property name="title">
    <string>Contacts Functions</string>
   </property>
   <widget class="QGroupBox" name="CreateContactBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Create Contact</string>
    </property>
    <widget class="QPushButton" name="CreateContactButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Create Contact</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="UpdateContactBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Update Contact</string>
    </property>
    <widget class="QPushButton" name="UpdateContactButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Update Contact</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="DeleteContactBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Delete Contact</string>
    </property>
    <widget class="QPushButton" name="DeleteContactButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete Contact</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="EventsBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>300</width>
      <height>170</height>
     </rect>
    </property>
    <property name="title">
     <string>Events</string>
    </property>
    <widget class="QTableView" name="EventsTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>280</width>
       <height>140</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ContactsForTheEventBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>300</width>
      <height>170</height>
     </rect>
    </property>
    <property name="title">
     <string>Contacts For The Event</string>
    </property>
    <widget class="QTableView" name="ContactsForTheEventTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>280</width>
       <height>140</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>430</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="HorizontalLayout">
     <item>
      <spacer name="HorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
